src/plugins/auto/executable_network.cpp

```diff
@@ -504,6 +603,19 @@ bool MultiDeviceExecutableNetwork::ScheduleToWorkerInferRequest(Task inferPipeli
                 deviceInfo.deviceName = _loadContext[CPU].workName;
                 devices.push_back(std::move(deviceInfo));
             }
+
+            if (_context.performanceHint == PluginConfigParams::CUMULATIVE_THROUGHPUT) {
+                for (auto deviceInfo : _loadContext[ACTUALDEVICE].cumulativeDevices) {
```

Comment:
cumulativeDevices is not locked

Reply:
also please make sure the device order is kept as it was

Reply:
push ACTUALDEVICE; then push cumulativeDevices; at last if not exist CPU_HELP in devices  try to push CPU_HELP